### PR TITLE
Convert full framework targeted csproj to use SDK-style

### PR DESCRIPTION
**Pull request by imcarolwang**
Update below full framework target csproj to use SDK style format:
- CertificateGenerator.csproj 
- SelfHostedWCFService.csproj

### CODE REVIEW

File: src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj [hunk 1]
```diff
@@ -1,36 +1,14 @@
...
     <OutputType>Exe</OutputType>
-    <RootNamespace>CertificateGenerator</RootNamespace>
-    <AssemblyName>CertificateGenerator</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFramework>net471</TargetFramework>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="BouncyCastle.Crypto.dll" Version="1.8.1" GeneratePathProperty="true" />
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.2" PrivateAssets="All" />
```
Comment (ViktorHofer) [inline src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj:10]: You don't need this package anymore. It's automatically included.

Comment (imcarolwang) [reply src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj:10]: I tested it offline, the result shows that without referencing package `Microsoft.NETFramework.ReferenceAssemblies`, the below build command got "_error MSB3644: The reference assemblies for .NETFramework,Version=v4.7.1 were not found._"  
Did I miss something?

`eng\common\cibuild.cmd -warnAsError 0 -configuration Release -projects src\System.Private.ServiceModel\tools\CertificateGenerator\CertificateGenerator.sln /p:Sign=false`

Comment (ViktorHofer) [reply src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj:10]: I'll take a look. Don't treat this feedback as blocking meanwhile.

File: src/System.Private.ServiceModel/tools/scripts/BuildCertUtil.cmd [hunk 2]
```diff
@@ -1,41 +1,14 @@
...
-set VSCMD_START_DIR="%~dp0"
-call "%_VSCOMNTOOLS%\VsDevCmd.bat"
-goto :EnvSet
-
-:MissingVersion
-:: Can't find VS 2017, 2019
-echo Error: %~nx0 requires Visual Studio 2017 or 2019.
-echo        Please see https://github.com/dotnet/wcf/wiki/Developer-Guide for build instructions.
-exit /b 1
+@echo on
```
Comment (mconnew) [inline src/System.Private.ServiceModel/tools/scripts/BuildCertUtil.cmd:2]: I presume this was turned on for debugging purposes. Unless you have a specific reason to leave echo on, it should be turned off.

File: src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj (hunk 1 above)
Comment (ViktorHofer) [reply src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj]: @imcarolwang I noticed that you introduced a PackageReference to ConfigurationManager which isn't necessary as `System.Configuration` is already inbox in .NET Framework and can just be directly referenced. The ConfigurationManager package is just a full facade that type forwards to System.Configuration. I pushed to your branch to clean that up. I hope that's OK.

Regarding this issue, I couldn't reproduce the error. Can you please try this again?

File: src/System.Private.ServiceModel/tools/scripts/BuildCertUtil.cmd (hunk 2 above)
Comment (imcarolwang) [reply src/System.Private.ServiceModel/tools/scripts/BuildCertUtil.cmd:2]: I see. You're right it's turning on for debug only. Turning off.

File: src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj (hunk 1 above)
Comment (imcarolwang) [reply src/System.Private.ServiceModel/tools/CertificateGenerator/CertificateGenerator.csproj]: @ViktorHofer Thank you for your help! I pull the code and retry, it works perfectly. I have no idea why it failed consistently last time. Now it works consistently :) Thanks again!

**Review by ViktorHofer — APPROVED** [review]

**Pull request merged**